app/routes/team.tsx

```diff
@@ -33,7 +32,13 @@ export default function Team() {
     for (const [idx, tutor] of all.entries()) {
         aboutUsCards.push(<section key={`tutor${idx}`} className="my-2 md:my-8 grid grid-flow-row gap-x-2 gap-y-2 md:gap-y-8">
         <div className="bg-white order-last md:order-first rounded-[3rem] md:rounded-[3rem] overflow-hidden">
-            <img src={tutor.img} alt="" width="100%"/>
+            <div className="image-container" style={{ width: '100%', height: '500px', overflow: 'hidden'  }}>
+          <img
+            src={tutor.img}
+            alt=""
+            style={{ objectFit: 'cover', width: '100%', height: '100%' }}
```

Comment:
Do you really need width 100% and height 100% here? Wont the image automatically resize?

Reply:
I thought this as well, but it appears that setting `width: '100%', height: '100%'` is necessary to make sure the images fully fill out the entire container. Some of the images also appear noticeably misaligned without this.
I put together an image showing the difference: https://ibb.co/42yy83j